src/renderer/Generics/redesign/CopyButton/copyButton.css.ts

```diff
@@ -0,0 +1,28 @@
+import { style } from '@vanilla-extract/css';
+import { vars } from '../theme.css';
+
+export const copyIcon = style({
+  color: vars.color.font40,
+  cursor: 'pointer',
+  selectors: {
+    '&:hover': {
+      color: '#753bf4',
```

Comment:
can be replaced with `vars.color.purple600`

Reply:
`common.color.purple600`